lib/services/waterline.js

```diff
@@ -125,8 +129,15 @@ function WaterlineServiceFactory(
         .then(function() {
             //it's OK to create indexes multiple times, so the code isn't protected by initialized
             //checking.
+            //
+            //Catching errors is because:
+            //  (1) When it fails to create indexes, the service still works.
```

Comment:
So does it mean that if the error is catched, and the service could continue to work?